api/flaskr/wcapi.py

```diff
@@ -0,0 +1,12 @@
+from woocommerce import API
+import os
+
+
+class woocommerce:
+    wcapi = API(
+        url="https://wooatasoy.wpcomstaging.com",
+        consumer_key = os.getenv("WOO_CONSUMER_KEY"),
+        consumer_secret = os.getenv("WOO_CONSUMER_SECRET"),
```

Comment:
Would you like to move this into user information into mongodb? Maybe we can even  have a functional integrations page. ( So user can paste their keys / secrets there and will be saved into db )

Reply:
Ok, I'll create another issue and modify the integrations page to let the user log their consumer keys. It'll be an enhancement to the Integrations page.